src/makeRobotAccountant.js

```diff
@@ -6,7 +6,25 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  const getSum = (a) => {
+    const add = (b) => {
+      if (count === 3) {
```

Comment:
The condition to check the count value is incorrect. It should be count > 3 and count is even, not count === 3.